### PR TITLE
Add Github Actions for Lint and Build

### PR DESCRIPTION
**Pull request by racerxdl**
Por enquanto ele só gera os artefatos e faz o lint.

Olha o #4  pra ver como tá as mensagens (olha a ultima mensagem só)

### CODE REVIEW

File: event/uuid.proto [hunk 1]
```diff
@@ -2,6 +2,9 @@ syntax = "proto3";
 
 package event;
 
+
+option java_outer_classname = "UUIDOuterClass";
```
Comment (airtonjal) [inline event/uuid.proto]: Is this used to avoid name clashing?

Comment (racerxdl) [reply event/uuid.proto]: Yes, it doesn't build without it. I think the protoc gen builds for java using one .proto file per class , so it names the class as Event instead UUID.

Comment (racerxdl) [reply event/uuid.proto]: Not sure if that's the correct way to fix it though.

**Comment by airtonjal** [comment]
Ficou muito bom @racerxdl !
[](http://www.sheawong.com/wp-content/uploads/2013/08/keephatin.gif)

**Pull request merged**